src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -57,6 +59,14 @@ public IPackageInstaller PackageInstaller
             get { return dynamoViewModel.PackageManagerClientViewModel; }
         }
 
+        /// <summary>
+        /// A reference to package loader to query for certain packages info
+        /// </summary>
+        public IPackageLoader PackageLoader
+        {
+            get { return dynamoViewModel.Model.GetPackageManagerExtension().PackageLoader; }
```

Comment:
This property is being obtained from a specific extension, namely, the PM extension in this case, still `PackageLoader` is being made generically available to all extensions. It seems a little counter-intuitive, no?

Reply:
I don't get the need for adding the `IPackageLoader` interface since `PackageLoader` type is accessible here.

Reply:
![image](https://user-images.githubusercontent.com/3942418/62894595-0197c100-bd1b-11e9-9bca-19fbf121f4cd.png)

This is so extension author will not get other members of package loader

Reply:
@aparajit-pratap @mjkkirschner Let me know what you think on this one, otherwise this PR is good for review with tests

Reply:
We can directly access `LocalPackages` in the `WorkspaceDependencyViewExtension` if that is what is needed by exposing it from `ReadyParams`:
`dynamoModel.GetPackageManagerExtension().PackageLoader.LocalPackages`?

Reply:
Yep, that works too, I think the idea with adding the interface is to start grouping some of these properties together, so that we don't have these parameter classes with a bunch of disorganized properties floating around.

Reply:
I still believe exposing package loader is more useful than local packages because packages list is pretty much useless for other extensions because they can't do anything other than list the info which are already done in `manage packages` menu item. But seems like you have a strong opinion on this @aparajit-pratap. I will remove the interface until we have a better less confusing solution.

Reply:
@QilongTang @aparajit-pratap - here is what I mean:
https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/Extensions/ViewStartupParams.cs#L13

you can access the ExtensionsManager from your extension, then get the packageManager, then get the packageLoader - without adding anything to ViewReadyParams.